weblate/templates/new-language.html

```diff
@@ -23,42 +23,42 @@
   <div class="panel-heading"><h4 class="panel-title">{% trans "Start new translation" %}</h4></div>
   <div class="panel-body">
 {% if object.new_lang == 'url' and not can_add %}
-<p><a href="{% url 'project' project=object.project.slug %}#information">{% trans "Information on how to start translating into another language is to be found in the translator instructions." %}</a></p>
+<p><a href="{% url 'project' project=object.project.slug %}#information">{% trans "The translator instructions has info on how to start translating into other languages." %}</a></p>
 {% else %}
 <p>
   {% trans "Please choose the language you want to translate to." %}
   {% if object.file_format_create_style == "create" %}
     {% if object.new_base %}
       {# Bilingual formats #}
-      {% blocktrans with new_base=object.new_base %}Translation file will be created based on <code>{{ new_base }}</code> with all strings being untranslated.{% endblocktrans %}
+      {% blocktrans with new_base=object.new_base %}A translation file will be created based on <code>{{ new_base }}</code> with all strings untranslated.{% endblocktrans %}
     {% else %}
       {# Monolingual formats, for example JSON or Android #}
-      {% trans "Translation file will be created blank and new strings will be added as they are translated." %}
+      {% trans "An empty translation file will be created and new strings added as they are translated." %}
     {% endif %}
   {% elif object.file_format_create_style == "copy" %}
     {# Documents, for example ODT, HTML, IDML #}
     {% if object.new_base %}
-      {% blocktrans with new_base=object.new_base %}Translation file will be created as a copy of <code>{{ new_base }}</code> with all strings marked as needing editing and strings will be replaced as they are translated.{% endblocktrans %}
+      {% blocktrans with new_base=object.new_base %}A translation file will be created as a copy of <code>{{ new_base }}</code> with all strings marked as \"Needs editing\" and replaced as they are translated.{% endblocktrans %}
     {% else %}
       {# Should not happen #}
       Create style: {{ object.file_format_create_style }}
     {% endif %}
   {% elif object.file_format_create_style == "directory" %}
     {# App store metadata #}
-    {% trans "Empty directory will be created for translations and individual files will be created as the strings are translated." %}
+    {% trans "An empty directory will be created for translations, with individual files created as strings are translated." %}
   {% else %}
     {# Should not happen #}
     Create style: {{ object.file_format_create_style }}
   {% endif %}
 </p>
 {% if object.new_lang == 'contact' and not can_add %}
-<p>{% trans "Project maintainers are notified of this request, which prompts them to add the language manually." %}</p>
+<p>{% trans "Project maintainers are notified of this request, prompting them to add the language manually." %}</p>
```

Comment:
```suggestion
<p>{% trans "Project maintainers are notified of this request, prompting manual addition." %}</p>
```
Adding this as an example of why the original is tolerable.